Tasks/AzureResourceGroupDeployment/package.json

```diff
@@ -1,10 +1,7 @@
-
 {
   "name": "AzureResourceGroupDeployment",
-  "main":"main.js",
+  "main": "main.js",
   "dependencies": {
-    "vsts-task-lib" : "^0.9.20",
```

Comment:
Because these modules will be imported from common module. If I mentioned same module in common as well as in task it doesn't work properly.